_kits/it/progettazione-interfaccia.md

```diff
@@ -47,6 +47,10 @@ superAccordion:
       <p>Il kit è un progetto aperto e in continua evoluzione che permette, — grazie all’interazione con il team di sviluppo e la facile resa in codice con i kit di sviluppo web, in particolare con Boostrap Italia — di progettare e prototipare efficacemente l'interfaccia del touchpoint digitale in realizzazione.</p>"
       img: "/assets/icons/kit/kit_UI_InterfacciaUtente.svg"
       resources:
+        - title: Design system del Paese, nuovo UI kit v3.0.0-beta.1
+          icon: "/assets/icons/pencil.svg"
+          description: Versione completamente rinnovata del kit di progettazione interfaccia, caratterizzata da nuova tecnologia e implementazione design token. L'attuale versione beta, è usabile in produzione e coerente con Bootstrap Italia.
```

Comment:
```suggestion
          description: Versione 3.0.0-beta.1 del kit di progettazione interfaccia, caratterizzata da nuova tecnologia e implementazione design token. L'attuale versione beta è usabile in produzione e coerente con Bootstrap Italia
```

Reply:
@Fupete eviterei di scrivere "nuovo" "rinnovato" etc. perché adesso è appena uscito ma fra 6 mesi non sarà più così nuovo insomma....ti ho proposto una versione alternativa dell'incipit, inoltre mi piacerebbe esplicitare che tipo di tecnologia è invece di dire "nuova" appunto, se si può dirlo in maniera un minimo comprensibile. Ho tolto il punto alla fine perché in queste descrizioni non lo mettiamo mai

Reply:
+1 @danielaiozzo

Reply:
allora lascerei come da suggestion